tool_collections/kraken/.shed.yml

```
@@ -11,7 +11,7 @@ long_description: |
   and a novel classification algorithm.
 name: kraken
 owner: devteam
-remote_repository_url: https://github.com/galaxyproject/tools-iuc/blob/master/tool_collections/kraken/
+remote_repository_url: https://github.com/galaxyproject/tools-iuc/tree/master/tool_collections/kraken/
```

Should we also update `master` to `main` on all these?

```suggestion
remote_repository_url: https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/kraken
```